### ISSUE TITLE
Incorrect HTML entity in Dutch translation – “Sample met getoetste webpagina&#39;s”

### ISSUE THREAD

**Issue by HypeHenry** [w3c/wai-wcag-em-report-tool#231]
In the Dutch translation of the WCAG-EM Report Tool, an incorrect HTML entity appears in the text:

Sample met getoetste webpagina&#39;s

The entity &#39; is displayed instead of a proper apostrophe. The correct text should be:
Sample met getoetste webpagina’s

Location:
When downloading the report in HTML format (Dutch language version, nl).

Expected result:
Sample met getoetste webpagina’s

Actual result:
Sample met getoetste webpagina&#39;s
